.gitignore

```diff
@@ -0,0 +1,136 @@
+# Jupyter Book
```

Comment:
This .gitignore file has lines from its parent copy in jupyter/accessibility that are not needed here, but I might be too lazy to update it.

Reply:
ah no harm leaving there